index.html

```diff
@@ -5,55 +5,108 @@
+                    <label class="slider-controls-item slider-controls-item-1" for="slider-1">
+                        <span class="visually-hidden">Первый</span>
+                    </label>
+                    <label class="slider-controls-item slider-controls-item-2" for="slider-2">
+                        <span class="visually-hidden">Второй</span>
+                    </label>
+                    <label class="slider-controls-item slider-controls-item-3" for="slider-3">
+                        <span class="visually-hidden">Третий</span>
+                    </label>
+                </div>
+            </div>
+        </div>
+    </div>
     <header>
-        <nav>
-            <a>
-                <img src="svg/logo.svg" alt="Логотип Gllacy" width="154" height="64">
-            </a>
-            <ul>
-                <li><a href="#">Каталог</a></li>
-                <li><a href="#">Доставка и оплата</a></li>
-                <li><a href="#">О Компании</a></li>
+        <a class="logo">
+            <img src="svg/logo.svg" alt="Логотип Gllacy" width="154" height="64">
+        </a>
+        <nav class="main-navigation">
+            <ul class="site-navigation">
+                <li class="sub-menu">
+                    <a href="#">Каталог</a>
+                    <ul class="main-navigation-sub">
+                        <li class="sub-menu-title">Новинки</li>
+                        <li>
+                            <a href="#">Сливочное</a>
+                        </li>
+                        <li>
+                            <a href="#">Щербеты</a>
+                        </li>
+                        <li>
+                            <a href="#">Фруктовый лёд</a>
+                        </li>
+                        <li>
+                            <a href="#">Мелорин</a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="navigation-item">
+                    <a href="#">Доставка и оплата</a>
+                </li>
+                <li class="navigation-item-1">
+                    <a href="#">О Компании</a>
+                </li>
             </ul>
+        </nav>
+
+        <div>
             <ul>
-                <li>
-                    <button type="button">Поиск</button>
+                <li class="search">
+                    <form class="search-form" action="#" method="post">
+                        <input type="search" name="search" id="search" placeholder="Что ищем?">
+                        <button type="submit">Поиск</button>
+                    </form>
                 </li>
                 <li>
                     <button type="button">Вход</button>
+                    <div class="block">
+                        <form class="modal-form" action="#">
+                            <input type="text" name="email" placeholder="email@example.com"><br>
+                            <input type="password" name="password" placeholder="••••••••"><br>
```

Comment:
```suggestion
                            <input type="text" name="email" placeholder="email@example.com">
                            <input type="password" name="password" placeholder="••••••••">
```
не нужно это дерьмо тебе тут, везде где ты думаешь что нужен <br> в 99.9 случаев можно избежать его, и лучше это делать